.circleci/config.yml

```diff
@@ -88,6 +88,33 @@ jobs:
           command: |
             source lint-env/bin/activate
             black --check cla_backend
+  pip-compile:
+    docker:
+      - image: circleci/python:2.7
```

Comment:
I think this image might be deprecated in favour of cimg images. [https://circleci.com/developer/images/image/cimg/python](url)
[https://discuss.circleci.com/t/legacy-convenience-image-deprecation/41034](url)

Reply:
Good spot,I have now changed this to use the cimg image